### ISSUE TITLE
Add focus management

### ISSUE THREAD

**Issue by alaindet** [alaindet/date-pickle#14]
Focus management is important for accessibility concerns. In particular, these features should be added

Date Picker
- Focus the previous weekday (moves to previous month if needed)
- Focus the next weekday (moves to next month if needed)
- Focus the same weekday of the previous week (moves to previous month if needed)
- Focus the same weekday of the next week (moves to next month if needed)
- Focus the first day of the month
- Focus the last day of the month
